tests/nnstreamer_query/runTest.sh

```diff
@@ -132,6 +132,11 @@ _callCompareTest raw8_2.log result8_2.log 8-5 "Compare 8-5" 1 0
 kill -9 $pid &> /dev/null
 wait $pid
 
+# Skip query-hybrid test until test failue is fixed. (Related issue: https://github.com/nnstreamer/nnstreamer/issues/3877)
+rm *.log
+report
+exit
+
```

Comment:
No. please simply update failing TCs optional.

Reply:
Then. later, you may turn the mosquitto service up and down in this script with the help from .spec file.

Reply:
Ok.. let's do it with this solution. LGTM